### PR TITLE
Add gracedb-sdk

### PR DESCRIPTION
**Pull request by duncanmmacleod**
This PR adds recipes for the following packages:

- `pytest-freezegun`
- `pytest-httpserver`
- `ligo-requests`
- `gracedb-sdk`

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by duncanmmacleod** [comment]
@conda-forge/help-python, I think this is ready, but the tests fail on windows because it uses `os.getuid`. Should I remove the `noarch: python`?

**Comment by duncanmmacleod** [comment]
Ping @conda-forge/help-python, @conda-forge/staged-recipes, I think this is ready, but the tests fail on windows because it uses `os.getuid()`. Should I remove the `noarch: python`?

File: recipes/ligo-requests/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{% set name = "ligo-requests" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "5575ab76aa00ad175b1493ab53db809e1327f423fc8004798daf45bab1748bd5"
```
Comment (chrisburr) [inline recipes/ligo-requests/meta.yaml]: I think the update bot doesn't work if you quote the checksums:
```suggestion
  sha256: 5575ab76aa00ad175b1493ab53db809e1327f423fc8004798daf45bab1748bd5
```
The only bit that generally needs to be quoted is the version. It doesn't matter with `X.Y.Z` versioning but it does with two digits such as `1.1` and `1.10` as they get interpreted as YAML floats and the trailing `0` is dropped.

File: recipes/gracedb-sdk/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{% set name = "gracedb-sdk" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name|replace('-', '_') }}-{{ version }}.tar.gz"
+  sha256: "a4c68ea540ef0384c17a158d8c8dcc5ae3c28d05c94037ba9f0344d727dc2fd0"
```
Comment (chrisburr) [inline recipes/gracedb-sdk/meta.yaml]:
```suggestion
  sha256: a4c68ea540ef0384c17a158d8c8dcc5ae3c28d05c94037ba9f0344d727dc2fd0
```

File: recipes/pytest-freezegun/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
+{% set name = "pytest-freezegun" %}
+{% set version = "0.3.0.post1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip"
+  sha256: "b86b13ef75959bedf4c32f1fd81fec66fa4502d9892e0ef6ad1717a34fe1560e"
```
Comment (chrisburr) [inline recipes/pytest-freezegun/meta.yaml]:
```suggestion
  sha256: b86b13ef75959bedf4c32f1fd81fec66fa4502d9892e0ef6ad1717a34fe1560e
```

File: recipes/pytest-httpserver/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,55 @@
+{% set name = "pytest-httpserver" %}
+{% set version = "0.3.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name|replace('-', '_') }}-{{ version }}.tar.gz"
+  sha256: "c7269aed8b4f2d24e41eaa2034dd9ab7ae02b0cb6e71afeae93127832920ec8d"
```
Comment (chrisburr) [inline recipes/pytest-httpserver/meta.yaml]:
```suggestion
  sha256: c7269aed8b4f2d24e41eaa2034dd9ab7ae02b0cb6e71afeae93127832920ec8d
```

**Pull request merged**